README.md

```diff
@@ -19,7 +19,50 @@ This repository allows you to quickly install Apache Solr for Drupal 9+ into a [
      * Set "solr core" to `dev`
      * Under "Advanced server configuration" set the "solr.install.dir" to `/opt/solr`.
 
-1. `ddev restart`
+6. `ddev restart`
+
+## Installation on Silverstripe 4+
+
+1. `ddev get ddev/ddev-drupal9-solr && ddev restart`
+2. Install the required/relevant Silverstripe requirements: `ddev composer require firesphere/solr-search`
+    * Note that there are currently some tricky dependencies. This is known and to be fixed soon (tm)
```

Comment:
It's not fair to say this and at least not say something about it. Please link to it, or explain a bit, or something. And it doesn't really help to say "This is known and to be fixed soon (tm)" unless you include a date and the responsible party and a link to the issue where it will be taken care of.

Reply:
The responsible party for that issue is me, and I'm in the middle of resolving it.

The (tm) is because I'm not 100% sure if I can get around to it fully this month.

I'll update it in the documentation.

Reply:
Everybody knows about these things, but at least linking to the issue makes them know where to go for answers. Thanks!